.tavis.yml

```diff
@@ -0,0 +1,15 @@
+dist: xenial
+sudo: false
+
+language: generic
+
+env:
+  matrix:
+    - KONG_TEST_DATABASE=postgres
+    - KONG_TEST_DATABASE=cassandra
+
+install:
+  - make setup_tests
+  - cd kong-build-tools && make setup_tests

```

Comment:
This is pretty confusing.
A Makefile in the root repo and then in another repo which we are cloning too.
Is there a way we could simplify this please?

Reply:
yeah it's a sad pattern we've ended up with replicating in a few spots.

We could git-submodule which most folks seem opposed to or we could extract the common elements to a bash script and then
```
curl -fsSL https://raw.githubusercontent.com/Kong/kong-build-tools/${KONG_BUILD_TOOLS}/.ci/setup_ci.sh | bash
```